Vocable/Common/Views/ToastView.swift

```diff
@@ -22,6 +24,12 @@ class PhraseSavedView: BorderedView {
             cornerRadius = frame.height / 2
         }
     }
+    
+    var text: String = "" {
+        didSet {
+            titleLabel.text = text
+        }
+    }
 
     override func awakeFromNib() {
         super.awakeFromNib()
```

Comment:
```suggestion
        super.awakeFromNib()
        titleLabel.text = text
```
Ensuring that if the xib is loaded lazily, the titleLabel will get some text if it was already assigned